packages/libotf.rb

```diff
@@ -0,0 +1,44 @@
+require 'package'
+
+class Libotf < Package
+  description 'OpenType Font library'
+  homepage 'https://www.nongnu.org/m17n/'
+  @_ver = '0.9.16'
+  version @_ver
+  compatibility 'all'
+  source_url 'https://download.savannah.gnu.org/releases/m17n/libotf-#{@_ver}.tar.gz'
+  source_sha256 '68db0ca3cda2d46a663a92ec26e6eb5adc392ea5191bcda74268f0aefa78066b'
+
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libotf-0.9.16-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libotf-0.9.16-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libotf-0.9.16-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libotf-0.9.16-chromeos-x86_64.tar.xz'
+  })
+  binary_sha256({
+    aarch64: '48ea420876299f2de783c250b6def3c2a87e7bf8c16695a63b7561efbb9594f8',
+     armv7l: '48ea420876299f2de783c250b6def3c2a87e7bf8c16695a63b7561efbb9594f8',
+       i686: '487c98f04bdc12c06a2afd32087222150ee652d9fc3c6233daee341c41718406',
+     x86_64: '8ec1c9f4e8b99482323e3bf444cca95b5e3d0abb90b8ea278d1461b29470e340'
+  })
+
+  depends_on 'libxaw'
+  depends_on 'freetype'
+
+  def self.patch
+    system 'curl -Ls https://github.com/archlinux/svntogit-packages/raw/a67b940a19f1e7e47e7d8553ed31158ed70f1286/libotf/trunk/replace-freetype-config.patch | patch -p1 -b'
```

Comment:
Can we patch with a tagged release instead of trunk?

Reply:
Also, no sha256sum check?

Reply:
That's a tagged release, not trunk:
`https://github.com/archlinux/svntogit-packages/raw/a67b940a19f1e7e47e7d8553ed31158ed70f1286/libotf/trunk/replace-freetype-config.patch`

Not sure about the point of a sha256sum for a tagged git file downloaded from github via https? What is that trying to accomplish? Are we worried about curl being unable to download the file w/o corruption?

Reply:
Whenever you download anything using curl, wget or whatever, there is always the possibility the download will fail or only download partially.  This is just consistent with everything else we download in all our packages.  Should be SOP.  I checked out the https://github.com/archlinux/svntogit-packages/ repo and it doesn't look like they have any tagged releases yet so a git hash is fine.

Reply:
In general archlinux doesn't bother tagging releases, so git tags are the only option there.

Reply:
Using incoming hashpipe package now. :)